File: tests/go/cases/case_291_dead_sink_decoy_08.go
package escape_tests

var retainedCase291 = []map[string]string{}

func Case291DeadSinkDecoy08(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_sink_decoy_08",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "dead_sink_decoy_08:" + raw,
	}
	if 1 == 0 {
		retainedCase291 = append(retainedCase291, payload)
	}
	// SAFE: dead branch never executes sink.
	return payload["checksum"]
}

File: tests/nodejs/cases/case_291_dead_sink_decoy_08.js
/**
 * Task case 291: dead_sink_decoy_08 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case291DeadSinkDecoy08(input) {
  const taskName = 'dead_sink_decoy_08';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case291DeadSinkDecoy08 };
